JourneysSquad/JourneysSquad/View/AddBillView/LabelWithTextFieldView.swift

```diff
@@ -0,0 +1,70 @@
+import SwiftUI
+
+/// A text field that allows to enter and edit text and has a label above itself.
+struct LabelWithTextFieldView: View {
+    // MARK: - Internal interface
+
+    /// The text entered into the field.
+    @Binding var input: String
+
+    /// A label displayed above the field.
+    let label: String
```

Comment:
This is not a label, but text:

```suggestion
    /// A text displayed in the label.
    let labelText: String
```

Reply:
Please review the rest of code.

Reply:
Fixed